docs/root/configuration/listeners/network_filters/postgres_proxy_filter.rst

```diff
@@ -109,4 +109,10 @@ SQL statement was parsed successfully and metadata was created. The metadata is
   <table.db>, string, The resource name in *table.db* format.
   [], list, A list of strings representing the operations executed on the resource. Operations can be one of insert/update/select/drop/delete/create/alter/show.
 
-Parsing SQL statements and emitting metadata can be disabled by setting :ref:`enable_sql_parsing<envoy_v3_api_field_extensions.filters.network.postgres_proxy.v3alpha.PostgresProxy.enable_sql_parsing>` to false.
+.. attention::
+
+   Currently used parser does not successfully parse all Postgres SQL statements and it cannot be assumed that all SQL queries will successfully produce Dynamic Metadata.
```

Comment:
s/Postgres SQL/Postgres/

The project name is `PostgreSQL`, and `Postgres` is just an abbreviation, so although the grammar is correct it may sounds weird, no?

Reply:
Yes, it is indeed confusing. Corrected a bit this sentence.